protocol/ipnetwork.go

```diff
@@ -24,11 +24,27 @@ type IPNetwork struct {
 	Port43
 }
 
+type ReverseDS struct {
+	Zone       string  `json:"zone"`
+	KeyTag     int     `json:"keyTag"`
+	Algorithm  int     `json:"algorithm"`
+	Digest     string  `json:"digest"`
+	DigestType int     `json:"digestType"`
+	Events     []Event `json:"events,omitempty"`
+}
+
+type ReverseDelegationSecureDNS struct {
+	ZoneSigned       *bool       `json:"zoneSigned,omitempty"`
```

Comment:
Does it really need to be a pointer?

Reply:
I think so, this way the JSON tags can be omitted